docs/reference/gettingstartedexample/gse-debug-guide.md

```diff
@@ -0,0 +1,229 @@
+- If the **last byte flushed** stays false when a generation completes, the last packet will never have been generated.
+- If the **number of generated packets** does not match the expected number of generated packets, the data set might not be valid. This can happen if the data is expected to run faster than the serializer can output, or if the packets are getting stuck in DRAM.
+
+
+### Instrumenting and Monitoring FPGA Behavior
+When debugging or investigating FPGA behavior, there are some general strategies that you can implement to figure out where errors are coming from. You can monitor the boolean signals that are passed to the host through the FPGA front panel, or you can monitor various internal state machines. The exact nature of what you should instrument is dependent on the problem you are trying to solve, but there are three common strategies for getting debug information out of the LabVIEW FPGA image at runtime.
+    > Note: All of the strategies below will require the FPGA to be recompiled to reflect the suggested changes.
+
+#### Add additional indicators that allow you to poll and view the information from the host. 
+Adding indicators will allow you to read the status back of intersting signals from the host. This will give you a "last updated" view of the signals in question. It will not allow you to monitor every change to a signal but can give you good insights.
+![Generation In Reset](../../images/PXIe-148X-Gen-In-Reset.PNG)
+
+#### You can latch signals and add new indicators
+Dropping down a basic latch when a certain condition occurs can allow you to save a particular state. Make sure to wire up the reset signal so that you can clear your state. This will allow you to capture a specific change on the FPGA.
+![Latch Ready Indicator](../../images/PXIe-148X-Latch-Ready-Indicator.PNG)
+
+#### You can count rising and falling edges to see where things are going wrong
+This is a good way to measure boolean signals to see how many times a certain condition occurs. This can give you insights into complexe data flow problems wondering how many times a gate or state is allowing valid data to flow.
+![Latch Ready Counter](../../images/PXIe-148X-Latch-Ready-Counter.PNG)
+
+#### Add an FPGA to Host FIFO to send up debug information sideband.
+This technique takes a long time to setup, can dramatically change the resource utilization, and can change overall system bandwidth usage. It is the best way to instrument every single change on a cerain signal and pipe it up to the host. Having the host pull data from the fifo and store it can lead to bandwidth and file system concerns, so only use this technique if you really need to capture every change on a signal in the FPGA.
+![DMA Debug](../../images/PXIe-148X-DMA-Debug.PNG)
```

Comment:
I don't know why this only happens on this image, but the image doesn't end up on a new line.
![image](https://user-images.githubusercontent.com/6440492/198103147-50940a4d-8dec-46cd-b76a-897ab756e3ac.png)

Can you try adding a blank line after the text to see if that fixes it?  If not then I wouldn't worry about it. (Only mentioned it since there's a broken link to fix further up anyway...)

Reply:
Also, I fixed Github pages to again use your branch (in case you want to preview):

https://hernstrom.github.io/automotive-camera-module-reference/reference/gettingstartedexample/gse-debug-guide.html